packages/DatePicker/src/DatePicker.js

```diff
@@ -201,8 +206,6 @@ function DatePicker(props) {
     handleChange(selectedDate);
   }
 
-  const hasError = extendedInputProps && extendedInputProps.hasError;
```

Comment:
Shouldn't we also handle the `hasError` prop on `<DatePicker.Input />` ?

Reply:
Cause `hasError` is usually on `<Input />`

Reply:
Good point, Looks like we will need it on both now.

Reply:
Actually no the `hasError` prop is now automatically set on the Input at https://github.com/acl-services/paprika/blob/ux-681-datepicker-has-error-in-formelement-fix/packages/DatePicker/src/DatePicker.js#L237 so we don't need to include it as a prop on the `DatePicker.Input` this is a breaking api change though for DatePicker...

Reply:
discussed with @mikrotron Nicer to have a cleaner api, but for the time being we will retain the `hasError` prop on the `DatePicker.Input` component to avoid a breaking change.